README.md

```diff
@@ -5,3 +5,217 @@
 [![Build Status](https://dev.azure.com/trustbloc/edge/_apis/build/status/trustbloc.edge-service?branchName=master)](https://dev.azure.com/trustbloc/edge/_build/latest?definitionId=27&branchName=master)
 [![codecov](https://codecov.io/gh/trustbloc/edge-service/branch/master/graph/badge.svg)](https://codecov.io/gh/trustbloc/edge-service)
 [![Go Report Card](https://goreportcard.com/badge/github.com/trustbloc/edge-service)](https://goreportcard.com/report/github.com/trustbloc/edge-service)
+
+## Prerequisites (for running tests and demos)
+- Go 1.13
+- Docker
+- Docker-Compose
+- Make
+
+## Targets
+```
+# run checks and unit tests
+make all
+# run linter checks
+make checks
+# run unit tests
+make unit-test
+```
+
+#Dependency
+- [EDV](https://github.com/trustbloc/edv)
+
+# edge-service
+
+Edge Service is a reference implementation that demonstrates the following story:
+
+Issuer presents the data fetched from CMS, [edge-sandbox](https://github.com/trustbloc/edge-sandbox) for the following
+operations: 
+
+- Create issuer profile
+- Create verifiable Credential by using [aries-framework-go](https://github.com/hyperledger/aries-framework-go/tree/master/pkg/doc/verifiable) VC parser
+- Store verifiable credential in [EDV](https://github.com/trustbloc/edv)
+- Retrieve verifiable credential from c
+- Verify verifiable credential 
+
+## REST API ENDPOINTS 
```

Comment:
move it to subfolders like https://github.com/trustbloc/edge-sandbox/tree/master/docs